powerline/segments/common/bat.py

```diff
@@ -95,6 +95,14 @@ def _get_capacity(pl):
 	else:
 		pl.debug('Not using pmset: executable not found')
 
+	if sys.platform.startswith('cygwin'):
+		BATTERY_PERCENT_RE = re.compile('[0-9]+')
+		def _get_capacity(pl):
+			battery_summary = run_cmd(pl, ['WMIC', 'Path','Win32_Battery', 'GET', 'EstimatedChargeRemaining'])
```

Comment:
Invalid list formatting: comma must always be followed by one space or a line break.